packages/markerclusterer/package.json

```diff
@@ -20,16 +20,16 @@
     "agm"
   ],
   "peerDependencies": {
-    "@grupo-san-cristobal/agm-core": "14.0.0",
-    "@angular/common": "^14.0.0",
-    "@angular/core": "^14.0.0",
+    "@grupo-san-cristobal/agm-core": "15.0.0",
```

Comment:
I believe the caret is missed here. Getting an unnecessary peer dependency issue when installing the packages.